### ISSUE TITLE
[proposal] Добавить в VV лужи топлива ссылку на предмет из которого оно было вылито

### ISSUE THREAD

**Issue by Motzord** [TauCetiStation/TauCetiClassic#5391]
#### Подробное описание проблемы
Админ видит равномерно покрытый коридор в топливе и успешно не заметил логи брызганья топливом, админ недоволен и хочет узнать кто это сделал
#### Что должно было произойти
Админ открывает VV видит ссылку на огнетушитель и смотрит по последнему касанию
#### Что произошло на самом деле
Админ недоволен и идет смотреть демку/качать логи
#### Как повторить
Попытайтесь узнать кто разлил топливо 
#### Дополнительная информация:
Fire started это конечно хорошо, но чаще всего логируется бедный мимокрокодил с сигаретой, а не разливатель бензина

**Comment by LudwigVonChesterfield** [comment]
Нет. Закидывать ссылки в ВВ это мозгоебля с мусоросборщиком, а тем более когда это только для админов которым "лень скачать демку".

Может быть добавить название предмета из которого вылилось топливо - но это тоже очень бредово и тяжело будет реализовать, из-за того что там такая цепочка вызовов:

/*прок расскручивания/выливания у предмета*() --> reagents/react_turf() --> reagent/react_turf() --> new топливо.

Разве-что можно в последнем звене добавлять то, откуда топливо вылито. Но опять же, это как-то странно что фича делается исключительно для админов когда есть демки.

**Comment by Motzord** [comment]
> Нет. Закидывать ссылки в ВВ это мозгоебля с мусоросборщиком, а тем более когда это только для админов которым "лень скачать демку".
> 
> Может быть добавить название предмета из которого вылилось топливо - но это тоже очень бредово и тяжело будет реализовать, из-за того что там такая цепочка вызовов:
> 
> /_прок расскручивания/выливания у предмета_() --> reagents/react_turf() --> reagent/react_turf() --> new топливо.
> 
> Разве-что можно в последнем звене добавлять то, откуда топливо вылито. Но опять же, это как-то странно что фича делается исключительно для админов когда есть демки.

А если тогда добавить логирование выливания топлива/форона в секрет-панели?

**Comment by LudwigVonChesterfield** [comment]
Что?

**Comment by cool20141** [comment]
В логах пишется же, что, кто разлил

**Comment by Motzord** [comment]
> Что?

У педалек есть кнопочка bombing_list где написаны все что бахало и кто последний это трогал, можно было бы допилить, чтобы еще записывались все разливания топлива и кто что трогал.
